src/plugins/irc-events/away.js

```diff
@@ -25,4 +46,9 @@ module.exports = function(irc, network) {
 			user.away = away;
 		});
 	});
+
+	irc.on("back", (data) => {
```

Comment:
Does this actually work with away-notify?

It seems that the irc-fw code changed but we never updated it here.

Reply:
Yes.
![image](https://user-images.githubusercontent.com/2070503/41161346-d5eb79ce-6b32-11e8-9545-534e1088945d.png)